test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAnAppWithTransitiveProjectRefs.cs

```diff
@@ -28,7 +28,7 @@ public void It_builds_the_project_successfully()
             // (TestApp transitively depends on AuxLibrary)
 
             var testAsset = _testAssetsManager
-                .CopyTestAsset("AppWithTransitiveProjectRefs")
+                .CopyTestAsset("AppWithTransitiveProjectRefs", "BuildAppWithTransitiveProjectRef")
```

Comment:
That's not an additional TestAsset, it's the "callingMethod" parameter, which is used as the name of the folder where the test assets are copied to.  By default it uses the calling method name, but there is a test in another class with the same name, so this serves to disambiguate them and avoid failures if those tests happen to be run concurrently.